lib/fizzy/stack.hpp

```diff
@@ -35,4 +37,65 @@ class Stack : public std::vector<T>
     /// Drops @a num_elements elements from the top of the stack.
     void drop(size_t num_elements = 1) noexcept { resize(size() - num_elements); }
 };
+
+class OperandStack
+{
+    /// The pointer to the top item, or below the stack bottom if stack is empty.
+    uint64_t* m_top;
```

Comment:
```suggestion
    uint64_t* m_top = nullptr;
```

Reply:
I didn't what to do that because `m_top` is never `nullptr` actually but always alias `m_storage`.
This would be nicer if order of fields reverted, but we want to keep `m_top` as first one because it is accessed the most (then `&m_top === this`).

Reply:
It would be useful to add a comment why we keep it the first.